jobsdb/jobsdb.go

```diff
@@ -2288,14 +2288,14 @@ func (jd *HandleT) doStoreJobsInTx(tx *sql.Tx, ds dataSetT, jobList []*JobT) err
 	return err
 }
 
-func (jd *HandleT) storeJob(tx *sql.Tx, ds dataSetT, job *JobT) (err error) {
+func (jd *HandleT) storeJob(ctx context.Context, tx *sql.Tx, ds dataSetT, job *JobT) (err error) {
```

Comment:
please adapt `Prepare` statements and use `PrepareContext` instead

Reply:
modified